subgraph.template.yaml

```diff
@@ -46,7 +46,7 @@ dataSources:
     source:
       address: '{{ address }}'
       startBlock: {{ startBlock }}
-      abi: {{ name }}
+      abi: MarketWETH
```

Comment:
why?

Reply:
for the record, we discussed and renamed it to just `Market`